core/src/main/java/org/apache/calcite/sql/fun/SqlLibraryOperators.java

```diff
@@ -233,7 +234,7 @@ private static SqlCall transformConvert(SqlValidator validator, SqlCall call) {
       };
 
   /** The "DECODE(v, v1, result1, [v2, result2, ...], resultN)" function. */
-  @LibraryOperator(libraries = {ORACLE, SPARK})
+  @LibraryOperator(libraries = {ORACLE, REDSHIFT, SPARK})
```

Comment:
Should you add tests to SqlOperatorTest?

Reply:
I have updated the `SqlOperatorTest` tests to also test for the REDSHIFT library where it makes sense.